docs/source/usersguide/install.rst

```diff
@@ -74,6 +74,306 @@ are no longer supported.
+provided in the CMakeLists.txt file. To see a list of these variants and other
+information:
+
+.. code-block:: sh
+
+    spack info openmc
+
+    CMakePackage:   openmc
+
+    Description:
+        The OpenMC project aims to provide a fully-featured Monte Carlo particle
+        transport code based on modern methods. It is a constructive solid
+        geometry, continuous-energy transport code that uses ACE format cross
+        sections. The project started under the Computational Reactor Physics
+        Group at MIT.
+
+    Homepage: http://openmc.readthedocs.io/
+
+    Tags: 
+        None
+
+    Preferred version:  
+        0.12.0     [git] https://github.com/openmc-dev/openmc.git at tag v0.12.0
+
+    Safe versions:  
+        develop    [git] https://github.com/openmc-dev/openmc.git on branch develop
+        master     [git] https://github.com/openmc-dev/openmc.git on branch master
+        0.12.0     [git] https://github.com/openmc-dev/openmc.git at tag v0.12.0
+        0.11.0     https://github.com/openmc-dev/openmc/tarball/v0.11.0
+        0.10.0     https://github.com/openmc-dev/openmc/tarball/v0.10.0
+
+    Variants:
+        Name [Default]                 Allowed values          Description
+        ===========================    ====================    ==================================
+
+        build_type [RelWithDebInfo]    Debug, Release,         CMake build type
+                                       RelWithDebInfo,         
+                                       MinSizeRel              
+        debug [off]                    on, off                 Enable debug flags
+        ipo [off]                      on, off                 CMake interprocedural optimization
+        mpi [off]                      on, off                 Enable MPI support
+        openmp [on]                    on, off                 Enable OpenMP support
+        optimize [off]                 on, off                 Enable optimization flags
+
+    Installation Phases:
+        cmake    build    install
+
+    Build Dependencies:
+        cmake  git  hdf5  mpi
+
+    Link Dependencies:
+        hdf5  mpi
+
+    Run Dependencies:
+        None
+
+    Virtual Packages: 
+        None
+
+.. note::
+
+    It should be noted that by default OpenMC builds with ``-O2 -g`` flags which
+    are equivalent to a CMake build type of `RelwithDebInfo`. In addition, MPI
+    is OFF while OpenMP is ON.
+
+Assuming one had configured Spack with a GNU 9.3.0 compiler, to build OpenMC
+with optimization, OpenMP support and OpenMPI, use:
+
+.. code-block:: sh
+
+    spack install openmc+mpi+optimize+openmp ^openmpi %gcc@9.3.0
+
+Although the OpenMP variant is by default ON, one can explicitly write it in
+the Spack specification. If a user wanted OpenMC without any MPI or OpenMP support, 
+the variants can be deactivated:
+
+.. code-block:: sh
+
+    spack install openmc~mpi+optimize~openmp
```

Comment:
Personally, I don't see a good reason for a user to turn OpenMP off and think we should omit this

Reply:
done